### DISCUSSION TITLE
How to define the Reynolds number of a rotor

### DISCUSSION THREAD

**Discussion by phsag** [byuflowlab/FLOWUnsteady#152] [Q&A] [answered]
Hi，
can I define a blade with Reynolds number？In the Rotor in Hover example,the Airfoil is defined with Aero file.How is the Reynolds number defined in the Aero file?
![QQ图片20240304165644](https://github.com/byuflowlab/FLOWUnsteady/assets/129523028/1b66de5d-ed13-4b8b-bafa-754e3608c0a7)

Thank you in advance !

**Comment by EdoAlvarezR** [comment, marked as answer]
Polars at the Reynolds numbers of the specific simulation are pre-computed automatically at the beginning of the simulation if you use `xfoil=true`. Otherwise, if you use `xfoil=false`, the simulation will read the prescribed airfoil polars specified in the `airfoils.csv` file, which are provided by the user (hence, it's up to the user to provide polars at the correct Reynolds number).
